src/clarity-angular/dropdown/_dropdown.clarity.scss

```diff
@@ -84,6 +84,18 @@ $dropdown-white: clr-getColor(lightest);
             visibility: hidden;
             z-index: map-get($clr-layers, dropdown-menu);
 
+            .dropdown-menu {
```

Comment:
This `.dropdown-menu` is inside another `.dropdown-menu`? I am a bit confused about how this is used in the template.